### DISCUSSION TITLE
im connecting thru a proxie.

### DISCUSSION THREAD

**Discussion by Joec6** [rejetto/hfs#580] [Q&A]
I hve to use a proxy. when i first set up hls i can connect on my local host. when i use my domain it will connect but the files section says connection error. if i sign in and click on retrieve list the list of files are there. i saw a message on the admin screen about a proxy attempting to access the server and to change the setting to 1 so i did that but still no love. any ideas?
wimdows os and localxpose.io

**Comment by rejetto** [comment]
i don't understand how can you get "connection error" but still sign in.
maybe a video, or a few screenshots, can help understanding.

**Comment by rejetto** [comment]
i saw you replies, as i get email notifications.
i guess you deleted them because you solved, or because you didn't want to make your address public?

**Comment by Joec6** [comment]
Ya the address thing. I’m not to bright sometimes. I have not solved it yet.
Thanks for responding.

**Comment by Joec6** [comment]
I sent hem to your email also

**Comment by rejetto** [comment]
i didn't get your email, but i saw the old posts of yours.
i tried to access your server (you'll find some access from Italy, it's me) and the response was not as expected.
I think your proxy is messing with data.
The browser is sending the request for the list, and it includes "Accept:text/event-stream", but HFS is answering like this information is not there. The most likely thing is that your proxy is removing/overwriting this information, preventing HFS from working correctly.
I can do nothing about it. You should ask the service provider.

**Comment by Joec6** [comment]
THANKS FOR LOOKING INTO IT.
APPRECIATE YOUR TIME
